src/server.c

```diff
@@ -5564,7 +5564,7 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
                 "aof_base_size:%lld\r\n"
                 "aof_pending_rewrite:%d\r\n"
                 "aof_buffer_length:%zu\r\n"
-                "aof_pending_bio_fsync:%llu\r\n"
+                "aof_pending_bio_fsync:%lu\r\n"
```

Comment:
our atomic vars (atomicvar.h) have many different implementations (depending on what's supported), in order to use them properly, it's not sufficient to just flag the variable as redisAtomic (which would only do the job if compiled with C11), you must also access it with the provided macros.
if we want to rely on that, maybe we need to do so in all the places that access it.

IMHO a volatile 32 bit variable is enough for our purpose.
volatile in this case just means that the compiler actually reads from the memory rather than cache it in registers, it doesn't mean anything multi-thread memory barriers.

Reply:
It was rather naive for me just to use atomicvar declaration and assume that we are working only with c++11 support.

I see your point regarding volatile. The "Do not use volatile except in low-level code that deals directly with hardware", has its exceptions. Especially without c++11 Atomic support. 

I will change it to volatile.